modules/virt-creating-vddk-image.adoc

```diff
@@ -3,33 +3,32 @@
 // * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
 
 [id="virt-creating-vddk-image_{context}"]
-= Creating a VDDK image
+= Creating and using a VDDK image
 
-You can convert the VMware Virtual Disk Development Kit (VDDK) into an image, and push the image to an image registry. Then, add the image to the `v2v-vmware` ConfigMap.
+You can download the VMware Virtual Disk Development Kit (VDDK), build a VDDK image, and push the VDDK image to your image registry. You then add the VDDK image to the `v2v-vmware` ConfigMap.
 
 .Prerequisites
 
-* You must configure an image registry.
-* You must download the VDDK archive file.
+* You must have access to an {product-title} internal image registry or a secure external registry.
 
 .Procedure
 
+. In a browser, navigate to link:https://www.vmware.com/support/pubs/[VMware Documentation].
+. Click *VMware SDK & API Product Documentation* -> *VMware Virtual Disk Development Kit (VDDK)*.
```

Comment:
Just wondering, is there any reason not to link directly to the https://www.vmware.com/support/developer/vddk/ page? I think I'd be more worried about navigation changing than from a broken link (which we do actually have checkers for).

Reply:
Oops. Thanks for catching this. That's an old link. VMware moved their SDKs to a differnt page. I'll change that

Reply:
BTW, that link won't work because it leads to documentation. They need to get closer to the download site, which is code.vmware.com